src/Web/Avalonia.Web.Blazor/AvaloniaView.razor

```diff
@@ -10,10 +10,42 @@
      onkeydown="@OnKeyDown"
      onkeyup="@OnKeyUp">
     
-    <canvas @ref="_htmlCanvas" @attributes="AdditionalAttributes"/>
-    
-    <input @ref="_inputElement"
-           class="overlay"
-           type="text"
-           oninput="@OnInput"/>
+    <canvas id="htmlCanvas" @ref="_htmlCanvas" @attributes="AdditionalAttributes"/>
+
+	<div id="nativeControlsContainer" @ref="_nativeControlsContainer" />
+
+    <input id="inputElement" @ref="_inputElement" type="text" oninput="@OnInput"/>
 </div>
+
+<style>
+#htmlCanvas {
```

Comment:
Moved from sample project. It makes sense to keep these styles in the core package instead.